### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the toast component.
The U.I snippet is attached below

image

### CODE REVIEW

File: Components/Toast/styles.css [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+.navigation-menu
+{
+    padding: 1.5rem 1rem;
+    /* width: 100%; */
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    align-items: center;
+    /* border-bottom: 2px solid var(--primary-color); */
```
Comment (Prabhav-Sharma) [inline Components/Toast/styles.css:37]: Remove comments that are not needed

File: Components/Toast/styles.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+{
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.sub-texts
+{
+    margin: 0.2rem;
```
Comment (Prabhav-Sharma) [inline Components/Toast/styles.css:58]: Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website

File: Components/Toast/toast.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            <div class="component-heading">Toast</div>
+
+            <p class="info-text">Alerts are used to show alert messages to the user.It acts as an attention grabber</p>
+
+            <div class="component-sub-heading">Types Of Toasts</div>
+
+            <p>Alerts are available in 5 different styles.You need to include
+                these classes in the div tag.Choose the appropriate style according to your requirement
+                <b> alert primary, alert secondary, alert success, alert fail, alert warning (e.g. class="alert
+                    warning")</b>
```
Comment (adith-a-danthi) [inline Components/Toast/toast.html]: Closing tag is missing

**Review by adith-a-danthi — APPROVED** [review]
Looks Good! Try to use `prettier` to format your code, it'll help keep code well-formatted and indented.

**Pull request merged**